constants.ts

```diff
@@ -26,6 +26,6 @@ export const PAGE_REFERENCES = [
 export const SANITY_API_VERSION = '2022-10-25'
 
 // Your Shopify store ID.
-// This is your unique store URL (e.g. 'my-store-name.myshopify.com').
+// This is your unique store URL (e.g. 'https://admin.shopify.com/store/my-store-name/').
```

Comment:
I think this line should make it clear that the `SHOPIFY_STORE_ID` const should just be the last segment of the URL, not the full URL.

Reply:
@thebiggianthead Would this commented approach suffice for the intent of communication:

```
// This is your unique store (e.g. 'my-store-name' in the complete URL of 'https://admin.shopify.com/store/my-store-name/').
```

Reply:
Makes sense to me 😄